docs/subgraph-manifest.md

```diff
@@ -29,6 +31,7 @@ Any data format which has a well-defined 1:1 mapping with [IPLD Canonical Format
 | --- | --- | --- |
 | **kind** | *String | The type of data source. Possible values: *ethereum/contract*|
 | **name** | *String* | The name of the source data. Will be used to generate APIs in mapping, and also for self-documentation purposes |
+| **network** | *String* | For blockhains this describes which network the subgraph targets. For Ethereum this could be, for example, "mainnet" or "rinkeby". |
```

Comment:
Blockhains may be the next big thing but for now we're focusing on blockchains. 😉